Spatial.tscn

```diff
@@ -185,7 +185,7 @@ __meta__ = {
 
 [node name="ImportPackDialog" type="FileDialog" parent="Control/Dialogs"]
 margin_left = 289.0
-margin_top = 36.0
+margin_top = 60.0
```

Comment:
Moved this back but shrunk it down a bit so it didn't overlap the editor bar and burrito icon